prediction/src/algorithms/evaluation.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..'))
+sys.path.append('..')
+import numpy as np
+import pylidc as pl
+from config import Config
+from classify import trained_model
+from tqdm import tqdm
+from src.preprocess.lung_segmentation import get_z_range
+
+CONFIDENCE_THRESHOLD = 0.5
+
+
+def get_accuracy(TP, TN, FP, FN):
```

Comment:
would you be able to use more descriptive variable names?

Reply:
Sure. Do you maybe see any major flaws in the implemented logic of the whole file? I'm just asking because an accuracy of 10% seems pretty low to me.